app/src/main/java/org/glucosio/android/activity/ExternalLinkActivity.java

```diff
@@ -39,115 +40,109 @@
     }
+  }
 
-    private void initView() {
-        webView.setWebViewClient(new WebViewClient() {
-            public boolean shouldOverrideUrlLoading(WebView view, WebResourceRequest request) {
-                return true;
-            }
+  @Override public String extractTitle() {
+    return extractExtra(TITLE_KEY);
+  }
 
-        });
-        setupToolbar();
-    }
+  @Override public String extractUrl() {
+    return extractExtra(URL_KEY);
+  }
 
-    private void initPresenter() {
-        presenter = new ExternalViewPresenter(this, new BasicNetworkConnectivity(this));
-        presenter.onViewCreated();
+  private String extractExtra(String key) {
+    String extra = "";
+    if (getIntent().getExtras() != null) {
+      extra = getIntent().getStringExtra(key);
     }
+    return extra;
+  }
 
-    @Override
-    public void setupToolbarTitle(String link) {
-        if (toolbarTitle.containsKey(link)) {
-            setToolbarTitle(getString(toolbarTitle.get(link)));
-        }
-    }
+  @Override public void loadExternalUrl(String url) {
+    webView.loadUrl(url);
+  }
 
-    @Override
-    public String extractTitle() {
-        String title = "";
-        if (getIntent().getExtras() != null) {
-            title = getIntent().getStringExtra(TITLE_KEY);
-        }
-        return title;
-    }
-
-    @Override public String extractUrl() {
-        String url = "";
-        if (getIntent().getExtras() != null) {
-            url = getIntent().getStringExtra(URL_KEY);
-        }
-        return url;
-    }
-
-    @Override
-    public void loadExternalUrl(String url) {
-        webView.loadUrl(url);
-    }
-
-    @Override
-    public void showNoConnectionWarning() {
-        AlertDialog dialog = new AlertDialog.Builder(this)
-            .setMessage(R.string.warning_internet_connection_required)
+  @Override public void showNoConnectionWarning() {
+    AlertDialog dialog =
+        new AlertDialog.Builder(this).setMessage(R.string.warning_internet_connection_required)
             .setCancelable(false)
             .setPositiveButton(R.string.mdtp_ok, new DialogInterface.OnClickListener() {
-                @Override public void onClick(DialogInterface dialog, int which) {
-                    ExternalLinkActivity.this.finish();
-                }
+              @Override public void onClick(DialogInterface dialog, int which) {
+                ExternalLinkActivity.this.finish();
+              }
             })
             .create();
-        dialog.show();
-    }
+    dialog.show();
+  }
 
-    private void setupToolbar() {
-        if (getSupportActionBar() != null) {
-            getSupportActionBar().setDisplayHomeAsUpEnabled(true);
-            getSupportActionBar().setDisplayShowHomeEnabled(true);
-        }
+  private void setupToolbar() {
+    if (getSupportActionBar() != null) {
+      getSupportActionBar().setDisplayHomeAsUpEnabled(true);
+      getSupportActionBar().setDisplayShowHomeEnabled(true);
     }
-
```

Comment:
That is want I would name formatting issue. Why your code was not formatted before?